aws/react-bedrock3.py

```
@@ -142,6 +142,8 @@ def execute(self):
 def query(question, max_turns=3):
     i = 0
     bot = ChatBot(prompt)
+    if bot == None:
```

Ruff

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)